<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KAccountsView</class>
 <widget class="QWidget" name="KAccountsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>185</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="KMyMoneyAccountTreeView" name="m_accountTree"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="searchLayout">
       <item>
        <widget class="KLineEdit" name="m_searchWidget">
         <property name="showClearButton" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_collapseButton">
         <property name="toolTip">
          <string>Collapse all accounts in the list</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_expandButton">
         <property name="toolTip">
          <string>Expand all accounts in the list</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="m_hiddenCategories">
     <property name="text">
      <string>Note: Unused categories are not shown as selected by settings.</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer name="Spacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>367</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="m_totalProfitsLabel">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Total Profits:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KMyMoneyAccountTreeView</class>
   <extends>QTreeView</extends>
   <header>kmymoneyaccounttreeview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>m_collapseButton</sender>
   <signal>clicked()</signal>
   <receiver>m_accountTree</receiver>
   <slot>collapseAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_expandButton</sender>
   <signal>clicked()</signal>
   <receiver>m_accountTree</receiver>
   <slot>expandAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>378</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
